lib/block_view.go

```diff
@@ -6768,6 +6843,16 @@ func (bav *UtxoView) _connectCreateNFT(
 	if postEntry.IsNFT {
 		return 0, 0, nil, RuleErrorCreateNFTOnPostThatAlreadyIsNFT
 	}
+
+	// We can't encrypt unlockable content if Buy Now is enabled.
+	if txMeta.HasUnlockable && isBuyNow {
+		return 0, 0, nil, errors.Wrapf(RuleErrorCannotHaveUnlockableAndBuyNowNFT, "_connectCreateNFT: ")
+	}
+	// We can't sell an NFT for 0.
+	if isBuyNow && txMeta.MinBidAmountNanos == 0 {
+		return 0, 0, nil, errors.Wrapf(RuleErrorBuyNowMustHaveMinBidAmountNanos, "_connectCreateNFT: ")
+	}
```

Comment:
Quick question: What does this break? It seems fine to set an NFT up for sale to the first person who bids. I would let the frontends handle the validation around this.

Reply:
0 means cancel my bid